xbmc/video/VideoItemArtworkHandler.cpp

```diff
@@ -0,0 +1,472 @@
+/*
+ *  Copyright (C) 2005-2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "VideoItemArtworkHandler.h"
+
+#include "FileItem.h"
+#include "MediaSource.h"
+#include "ServiceBroker.h"
+#include "TextureDatabase.h"
+#include "filesystem/Directory.h"
+#include "guilib/LocalizeStrings.h"
+#include "music/MusicDatabase.h"
+#include "settings/Settings.h"
+#include "settings/SettingsComponent.h"
+#include "utils/FileExtensionProvider.h"
+#include "utils/FileUtils.h"
+#include "utils/URIUtils.h"
+#include "utils/Variant.h"
+#include "utils/log.h"
+#include "video/VideoDatabase.h"
+#include "video/VideoInfoScanner.h"
+#include "video/VideoInfoTag.h"
+#include "video/VideoThumbLoader.h"
+#include "video/tags/VideoTagLoaderFFmpeg.h"
+
+using namespace VIDEO;
+using namespace XFILE;
+
+namespace
+{
+//-------------------------------------------------------------------------------------------------
```

Comment:
Why not use doxygen blocks for this?

Reply:
Could you please guide me here, how would such a comment look exactly (syntax) and what will be the result in generated documentation. I introduced these comments only to ease orientation in the cpp file containing implementation of different classes - and I consider those classes 'internal' - that's why I put them in a cpp including there declarations, I don't see the need for any doxygen. But maybe I'm missing the point here?

Reply:
Maybe leave it as is as it's not intended as actual documentation